core/src/test/resources/org/apache/calcite/test/RelOptRulesTest.xml

```diff
@@ -11183,7 +11183,7 @@ LogicalProject(EXPR$0=[> SOME($0, {
 LogicalProject(DEPTNO=[$0])
   LogicalFilter(condition=[=($cor0.JOB, $1)])
     LogicalTableScan(table=[[CATALOG, SALES, DEPT]])
-})])
+})], variablesSet=[[$cor0]])
```

Comment:
This looks really confusing. We should print variables side-by-side with Project, and variables should go before fields names.

Reply:
fixed, I suppose